src/components/ActionButton/ActionButton.module.css

```diff
@@ -0,0 +1,4 @@
+.action-button {
+  width: 100px;
```

Comment:
Width like this is too hardcoded, let's use something like padding-left and padding-right

Reply:
Added padding and changed it to min-width. If the option is not set, the buttons will have different sizes, is this expected?